packages/core/src/FrameLoop.ts

```diff
@@ -159,9 +159,8 @@ export class FrameLoop {
         ? config.initialVelocity[i]
         : config.initialVelocity
 
-      const precision = config.precision || Math.abs(to - from) / 1000 || 0.005
-
-      console.log(precision)
+      const precision =
+        config.precision || Math.min(1, Math.abs(to - from) / 1000)
```

Comment:
### What it does

Today, precision is used to determine when to rest the spring, ie when velocity is close enough to `0` and current position is close enough to its target position. The higher the precision, the longer the spring comes to rest.

Precision is set by default to `0.005`.

However, animating `opacity` requires usually more precision than animating `translate`. In that case, `0.005` is fine for `opacity` (if you're animating `opacity` from `0` to `1`, the spring would come to rest is when its position is higher than `0.995` and smaller than `1.005`. However, animating a `translate` from `0 to 100` would rest when the position is higher than `99.995` and smaller than `100.005`. This amount of precision is probably exaggerated for a movement of that kind.

This commit makes it so that the precision of the animation would by default depend on the magnitude of the displacement. When animating from `0` to `1` the default precision would be `0.001`. When animating from `0` to `100`, the default precision would be `0.1`.

### Why it matters
The commit tries to rest the spring when no more movement is perceivable by the user. Currently and in some situations where the displacement is significant, `onRest` would fire with a noticeable delay vs the moment where the user _thinks_ the spring is at rest. This commit aims at reducing this perception.

### By the numbers
Animating a spring with `config: { tension: 120, friction: 12, mass: 1 }` from `-250` to `250` would originally take `~1450ms` before `onRest` was fired. With this commit, it only takes `~820ms`.